### PR TITLE
TL translations for basic-branching-and-merging.asc

### PR DESCRIPTION
**Pull request by josteinfrancois**

### CODE REVIEW

File: book/03-git-branching/sections/basic-branching-and-merging.asc [hunk 1]
```diff
@@ -139,38 +139,30 @@ index.html |    1 +
 1 file changed, 1 insertion(+)
 ----
 
-This looks a bit different than the `hotfix` merge you did earlier.
-In this case, your development history has diverged from some older point.
-Because the commit on the branch you're on isn't a direct ancestor of the branch you're merging in, Git has to do some work.
-In this case, Git does a simple three-way merge, using the two snapshots pointed to by the branch tips and the common ancestor of the two.
+Ito ay mukhang medyo iba sa `hotfix` na merge na ginawa mo kamakailan lamang. Sa kasong ito, ang kasaysayan ng iyong pag-develop ay humiwalay mula sa ilang mas matandang punto. Dahil ang commit sa branch kung saan nandoon ka ay hindi isang direktang ninuno ng branch na iyong mine-merge, ang Git ay dapat gumawa ng ilang trabaho. Sa kasong ito, ang Git ay gagawa ng isang simpleng three-way na merge, gamit ang dalawang snapshot na itinuro ng mga dulo ng branch at ang karaniwang ninuno ng dalawa.
```
Comment (mind-blast) [inline book/03-git-branching/sections/basic-branching-and-merging.asc]: I think you should change mine-merge   to mini-merge

File: book/03-git-branching/sections/basic-branching-and-merging.asc [hunk 2]
```diff
@@ -139,38 +139,30 @@ index.html |    1 +
...
 
-.Three snapshots used in a typical merge
-image::images/basic-merging-1.png[Three snapshots used in a typical merge.]
+.Tatlong snapshot na ginamit sa isang tipikal na merge
```
Comment (mind-blast) [inline book/03-git-branching/sections/basic-branching-and-merging.asc]: I think you should change Tatlong snapshot  to Tatlong mga snapshot

File: book/03-git-branching/sections/basic-branching-and-merging.asc [hunk 3]
```diff
@@ -139,38 +139,30 @@ index.html |    1 +
...
+Ngayong ang iyong trabaho ay naka-merge na, wala ka nang higit pang kailangan para sa `iss53` na branch. Maaari mo nang isara ang tiket sa iyong ticket-tracking na sistema, at burahin ang branch:
 
 [source,console]
 ----
 $ git branch -d iss53
 ----
 
 [[_basic_merge_conflicts]]
-==== Basic Merge Conflicts
+==== Batayan ng mga Salungatan sa Pag-merge
```
Comment (mind-blast) [inline book/03-git-branching/sections/basic-branching-and-merging.asc]: I think this is much better :  Paunang Pagsamahin ang mga Kasalungat

**Comment by mind-blast** [comment]
@josteinfrancois please re-check my suggested words and after that that a pull request again once all revisions are done, so the project manager will able to merge your contribution to the official repository.
Thank you.

File: book/03-git-branching/sections/basic-branching-and-merging.asc (hunk 1 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/basic-branching-and-merging.asc]: Will update based on your suggestion. Thank you.

File: book/03-git-branching/sections/basic-branching-and-merging.asc (hunk 2 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/basic-branching-and-merging.asc]: Will change that one in a bit.

File: book/03-git-branching/sections/basic-branching-and-merging.asc (hunk 3 above)
Comment (severinolorillajr) [reply book/03-git-branching/sections/basic-branching-and-merging.asc]: How about "Mga Pangunahing Salungatan sa Pag-Merge"?

Comment (mind-blast) [reply book/03-git-branching/sections/basic-branching-and-merging.asc]: @severinolorillajr i think it is much better one.

Comment (josteinfrancois) [reply book/03-git-branching/sections/basic-branching-and-merging.asc]: I am a bit confused with this one. Merge conflicts here refers to the conflicts during the merging process. I think "Paunang mga Kasalungat sa Pag-merge" would make sense in this context somehow?

Comment (mind-blast) [reply book/03-git-branching/sections/basic-branching-and-merging.asc]: @josteinfrancois i think it fits the title more.

Comment (josteinfrancois) [reply book/03-git-branching/sections/basic-branching-and-merging.asc]: I'll take shoganaii's on this one. Thank you.

**Comment by severinolorillajr** [comment]
@josteinfrancois , please tag me once you are done with the review updates.
So we can merge this.

**Comment by josteinfrancois** [comment]
@severinolorillajr I have finished committing the updates basing on @robin-ho and your suggestions.

**Pull request merged**